config/kubernetes/sig-docs/teams.yaml

```diff
@@ -288,6 +288,7 @@ teams:
     - Sea-n
     - tanjunchen
     - tengqm
+    - kinzhi
```

Comment:
You will have to make sure the list is still sorted.

Reply:
It has been adjusted, thank you